src/main/webapp/css/main.css

```diff
@@ -16,18 +16,104 @@
 
 /* Prevent the page from becoming too wide. */
 body {
+	background: #5433FF;  /* fallback for old browsers */
+  background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
```

Comment:
didn't know gradients were a thing that could happen in css! so cool

Reply:
Yeah! Pretty fun stuff, I used UGradient